src/_includes/_nav.html

```diff
@@ -15,10 +15,10 @@
         <li class="navbar__main-nav__link__outer">
           <a class="navbar__main-nav__link {{"active" if (base_route == "blog")}}" href="/blog">blog</a>
         </li>
-        {# <li class="navbar__main-nav__link__outer">
-          <a class="navbar__main-nav__link{{"active" if (base_route == "presentations")}}"
-            href="/presentations">slides</a>
-        </li> #}
+        <li class="navbar__main-nav__link__outer">

```

Comment:
Not hugely important at this stage, but I think the code of conduct can sit outside the main nav (especially as we add other sections)

Reply:
I think in the interim it definitely needs to be here especially because we don't have a footer or anywhere else to put it and I don't think it should be buried in the blog somewhere. Happy to re-evaluate it once we get more top nav though.